### DISCUSSION TITLE
how to i make explorerpatcher have a search bar with everything toolbar?

### DISCUSSION THREAD

**Discussion by Rezy774** [valinet/ExplorerPatcher#2433] [Q&A]
im on windows 11 build: 22621.2428.59.1 and explorerpatcher build: [22621.2428.59.1] and i would like a search bar with search everything 
![Uploading Screenshot 2023-11-12 105428.png…]()

**Comment by pyrates999** [comment]
Can't view your image.  Please upload it again.
